e2e-nodejs/group-hotwallet-with-blockhash/test-hotwallet-with-blockhash.mjs

```diff
@@ -117,6 +119,13 @@ export async function main() {
 
   console.log(sessionSigs);
 
+  // console.log(authSig);
+
+  // if (authSig.signedMessage.includes(TEST_BLOCKHASH)) {
+  //   // return fail("authSig doesn't contain the blockhash");
+  //   console.log("authSig doesn't contain the blockhash");
+  // }
+
```

Comment:
Why did we comment this out?

Reply:
`authSig` is undefined we can remove this code